Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -783,7 +783,10 @@ the IP addresses of native devices which have the default route on the host or
 have Kubernetes InternalIP or ExternalIP assigned. InternalIP is preferred over
 ExternalIP if both exist. To change the devices, set their names in the
 ``devices`` Helm option, e.g. ``devices='{eth0,eth1,eth2}'``. Each
-listed device has to be named the same on all Cilium managed nodes.
+listed device has to be named the same on all Cilium managed nodes. Alternatively
+if the devices do not match across different nodes, the wildcard option can be 
+used, e.g. ``devices=eth+``, which would match any device starting with prefix
+``eth``.
```

Comment:
Maybe add that if no device can be matched, then cilium-agent will try to perform the auto detection.

Reply:
Added.